src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/cart/index.yaml

```
@@ -1,93 +1,89 @@
 twig_hooks:
     hooks:
         'sylius_shop.cart.page':
```

```suggestion
        'sylius_shop.cart.index':
```
or
```suggestion
        'sylius_shop.cart.summary':
```
and adjust the name of the file